### DISCUSSION TITLE
Sun not following Ecliptic of date

### DISCUSSION THREAD

**Discussion by mashbudj** [Stellarium/stellarium#636] [Q&A] [answered]
Hi, I am studying precession and looking for ancient eclipses. I have found a few in the last couple thousand years, and the next couple thousand, but before about 4000 BC the sun seems to follow the ecliptic of J2000 not the ecliptic of Date! Since they differ by a degree or more it makes things difficult. The ecliptic of date looks correct, as the first point of Aries moves correctly, but the sun is off. Thanks.
![screen shot 2019-02-20 at 6 40 10 pm](https://user-images.githubusercontent.com/7852917/53132819-f9ff1880-353e-11e9-83a0-fb4226f80668.png)

**Comment by alex-w** [comment]
This is not exactly bug. By default VSPO87 is enabled and this theory has valid range for -4000..8000 for Earth with J2000.0 positions outside of range. DE431 has valid rangle -13000..17000 with J2000.0 positions outside of range.

**Comment by gzotti** [comment, marked as answer]
This is not a bug. You are extrapolating the model, with all consequences.
Stellarium User Guide, Appendix F. On Eclipses, section F.7. on planet positions, F.3. On the calendar, section F.8.  For installing DE43x, read section 5.5.3.
